include/manif/impl/se2/SE2.h

```diff
@@ -61,6 +61,8 @@ struct SE2 : SE2Base<SE2<_Scalar>>
   MANIF_COMPLETE_GROUP_TYPEDEF
   using Translation = typename Base::Translation;
   MANIF_INHERIT_GROUP_API
+  using Base::transform;
+  using Base::rotation;
```

Comment:
Unrelated to Rn thing no? Do you want this on the same PR?

Reply:
I had to remove those from a macro that is meant to apply to all groups but Rn does not have `rotation` of course. So yeah it belongs to the same load of changes to get Rn.